test/test_backward.py

```diff
@@ -16,6 +16,10 @@
 )
 from _common_test import SdpaContext, SdpaParams, SdpaContextFromNPZ, AOTRITON_TORCH_ONLY_USE_CPU
 
+@pytest.fixture()
+def torch_gpu(worker_id):
+    return int(worker_id[2:]) if worker_id != "master" else None
```

Comment:
What is a typical value of `worker_id`?

Reply:
Very weird values and not well documented. It is `gw0` `gw1` etc.